src/authn/AuthCache.ts

```diff
@@ -0,0 +1,31 @@
+import Authenticator from './Authenticator'
+import Token from './Token'
+
+// Default to 10 seconds less than expected expiry to give some wiggle room near the end
```

Comment:
Could you add a link to the node side constant that should correlate with this? 🙏

Reply:
That's a good idea